### PR TITLE
Added TouraineTech 20, France

### PR DESCRIPTION
**Pull request by CedricGatay**
Added info for Touraine Tech '20

### CODE REVIEW

File: _conferences/touraine_tech_20.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: "Touraine Tech '20"
```
Comment (JakeWharton) [inline _conferences/touraine_tech_20.md]: Do we need the `'20` as part of the name?

**Review by JakeWharton — COMMENTED** [review]
Are there Android talks? I browsed the program for this year and only saw web in the "frontend / mobile" category.

**Comment by CedricGatay** [comment]
It depends on what people submit but this is not an Android focused conference

File: _conferences/touraine_tech_20.md (hunk 1 above)
Comment (CedricGatay) [reply _conferences/touraine_tech_20.md]: I removed the year as it makes no sense in title

**Comment by cortinico** [comment]
This can probably be closed as there are no Android talks in the agenda right?

**Comment by vanniktech** [comment]
Plus the conference is over

**Comment by CedricGatay** [comment]
I'm closing it, sorry for the noise.

**Pull request closed without merging**